pushy/src/main/java/com/turo/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -82,6 +85,8 @@
     private static final String CONTENT_AVAILABLE_KEY = "content-available";
     private static final String MUTABLE_CONTENT_KEY = "mutable-content";
     private static final String THREAD_ID_KEY = "thread-id";
+    private static final String SUMMARY_ARGUMENT = "summary-arg";
```

Comment:
Trivial: can we append `_KEY` to these names for consistency?

Reply:
Added in ed6aed6.